bin/publish

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env ruby
```

Comment:
This file is fine- we still need to tag and push to master before we generate the release notes. See how this was done in the n2 gem. Maybe give some credit to Pezza on this file too - unless he got it from somehwere....